unitTests/ui/FileTypesTest.ts

```diff
@@ -0,0 +1,13 @@
+import { FileTypes } from '../../src/ui/Misc/FileTypes';
+
+export function FileTypesTest() {
+  describe('FileTypes', () => {
+    it('safelyBuildFileTypeInfo should prevent XSS attacks by espacing inputs', () => {
+      const unsafeInput = 'Foo><img src=y onerror=alert(document.cookie)>';
+      const info = FileTypes.safelyBuildFileTypeInfo('objecttype', unsafeInput, unsafeInput);
+
+      expect(info.icon).toBe('coveo-icon objecttype Foo&gt;&lt;img-src=y onerror=alert(document.cookie)&gt;');
```

Comment:
Very nitpicky, but what would be your thoughts on a regex pattern matching here instead, since all we're checking for is the presence of the arrow brackets. I think it would make the test less prone to being broken.

Reply:
I wanted to test with a real world example, I can test using underscore's escape method directly instead of typing it out, making it more safe. I don't think we should test the underscore's escape method itself thought